README.md

```diff
@@ -84,7 +84,9 @@ ftp_close_connection(ctxt::ConnContext)
 FTP(;host="", block=true, implt=false, ssl=false, ver_peer=true, act_mode=false, user="", pswd="", binary_mode=true)
 close(ftp::FTP)
 download(ftp::FTP, file_name::String, save_path::String="")
-upload(ftp::FTP, file_name::String, file=nothing)
+upload(ftp::FTP, local_name::AbstractString)
+upload(ftp::FTP, local_name::AbstractString, remote_name::AbstractString)
```

Comment:
If you were using the syntax in the Julia manual you could do `upload(ftp, local_name, [remote_name])`